solutions/subsets_solution.rb

```diff
@@ -1,3 +1,18 @@
+# Write a recursive method `subsets(array)` that 
+# returns all of the subsets of an array
+
+# example => subsets([1,2,3])
+# should return => []
+#  [], 
+#  [1], 
+#  [2], 
+#  [1, 2], 
+#  [3], 
+#  [1, 3], 
+#  [2, 3], 
+#  [1, 2, 3]
+# ]
+
 def subsets(arr)
   return [[]] if arr.empty?
```

Comment:
Can we change the solution to take in an argument `array` instead of `arr`, as this is what the problem description uses, and students copying over the solution will see it not pass.